codecov.yml

```diff
@@ -13,3 +13,14 @@ coverage:
         informational: false
         only_pulls: false
     patch: off
+
+ignore:
+  - "*.yml"
+  - "*.sh"
+  - "go.mod"
+  - "go.sum"
+  - ".gitignore"
+  - "Dockerfile"
+  - "*.Dockerfile"
+  - "integration/*"
```

Comment:
we likely don't want to measure test coverage on integration tests as that is overkill, we would end up writing unit tests for our integration tests which is generally not advisable.